lib/saml/url_service.rb

```diff
@@ -54,10 +71,10 @@ def idme_url
     end
 
     def signup_url
-      build_sso_url(LOA::IDME_LOA1) + '&op=signup'
+      @query_params[:op] = 'signup'
+      idme_url
     end
 
-    # verification operation is only if the user clicks identity verification via ID.me
```

Comment:
Not true. This is also used for the identity proofing part of logins for LOA3 users.